app/components/trainees/confirmation/course_details/view.rb

```diff
@@ -51,6 +51,11 @@ def require_age_range?
           !trainee.early_years_route?
         end
 
+        def require_course_type?
+          # Show course if draft, otherwise only show if not early years
+          trainee.draft? || !trainee.early_years_route?
```

Comment:
I think this is the wrong way round - in the card:

> "For all Early Years trainees in **draft**, the 'Course details'  component should only have rows for the **Course start date** and the **Course end date**"

Reply:
We should just block all friday commits. Fixed now